### PR TITLE
updated tags.  removed common tags and added tags that made sense

### PR DESCRIPTION
**Pull request by kensipe**
tags are used for search... they need to help the user discover services.

### CODE REVIEW

**Comment by kensipe** [comment]
@elingg looking for a review.  this is an unusual update to existing packages

File: repo/packages/M/marathon/5/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
       "url": "https://github.com/mesosphere/marathon/blob/master/LICENSE"
     }
   ],
-  "tags": ["mesosphere", "framework"],
+  "tags": ["init","long-running"],
```
Comment (elingg) [inline repo/packages/M/marathon/5/package.json]: nit need a space here

**Comment by elingg** [comment]
Took a quick look and it looks good to me. We would need to make sure the tags are matching.

**Comment by kensipe** [comment]
what do you mean on making sure the tags are matching?

File: repo/packages/M/marathon/5/package.json (hunk 1 above)
Comment (kensipe) [inline repo/packages/M/marathon/5/package.json]: good eye!

**Pull request merged**